infrastructure/code/sarif_utils.go

```diff
@@ -20,18 +20,18 @@ import "github.com/snyk/snyk-ls/application/config"
 
 func newCodeRequestContext() codeRequestContext {
 	unknown := "unknown"
-	orgName := unknown
+	orgId := unknown
 	if config.CurrentConfig().Organization() != "" {
-		orgName = config.CurrentConfig().Organization()
+		orgId = config.CurrentConfig().Organization()
```

Comment:
How do you know that this is always the ID? What happens if we put the org name into PublicID?

Reply:
This is a GAF feature, the config always returns the org id.

Reply:
That may be, but the user can overwrite it with a name.

Reply:
Just as a final summary on this thread. @bastiandoetsch and I looked at the code and thereby could resolve his concerns.